src/js/components/form-elements/v2/TextInput.jsx

```diff
@@ -14,16 +14,21 @@ const TextInput = ({
   disabled,
   errorMessage,
   placeholder,
+  id,
+  name,
   ...fieldProps
 }) => (
   <InputWrapper
     title={title}
     tooltip={tooltip}
     required={required}
     button={button}
+    inputId={id || name}
     errorMessage={errorMessage}
   >
     <input
+      id={id || name}
+      name={name}
```

Comment:
this anyway comes from the `...fieldProps` so I assume you only extracted this for a better readability?

Reply:
I extracted `id` and `name` to also pass it to InputWrapper component, we are doing the filedProps spread only on input